internal/service/ch/queries.go

```diff
@@ -439,6 +414,56 @@ func getAggQuery(aggArgs *model.AggregatedSignalArgs) (string, []any, error) {
 	return stmt, args, nil
 }
 
+func buildConditionList(fil *model.SignalFloatFilter) []qm.QueryMod {
+	if fil == nil {
+		return nil
+	}
+
+	var mods []qm.QueryMod
+
+	if fil.Eq != nil {
+		mods = append(mods, qmhelper.Where(vss.ValueNumberCol, qmhelper.EQ, *fil.Eq))
+	}
+	if fil.Neq != nil {
+		mods = append(mods, qmhelper.Where(vss.ValueNumberCol, qmhelper.NEQ, *fil.Neq))
+	}
+	if fil.Gt != nil {
+		mods = append(mods, qmhelper.Where(vss.ValueNumberCol, qmhelper.GT, *fil.Gt))
+	}
+	if fil.Lt != nil {
+		mods = append(mods, qmhelper.Where(vss.ValueNumberCol, qmhelper.LT, *fil.Lt))
+	}
+	if fil.Gte != nil {
+		mods = append(mods, qmhelper.Where(vss.ValueNumberCol, qmhelper.GTE, *fil.Gte))
+	}
+	if fil.Lte != nil {
+		mods = append(mods, qmhelper.Where(vss.ValueNumberCol, qmhelper.LTE, *fil.Lte))
+	}
+	if len(fil.NotIn) != 0 {
+		mods = append(mods, qm.WhereNotIn(vss.ValueNumberCol+" NOT IN ?", fil.NotIn))
+	}
+	if len(fil.In) != 0 {
+		mods = append(mods, qm.WhereNotIn(vss.ValueNumberCol+" IN ?", fil.In))
```

Comment:
should this be qm.WhereIn instead of qm.WhereNotIn?

Reply:
Good catch! Fixed on main.